harness/determined/pytorch/deepspeed/_deepspeed_context.py

```diff
@@ -272,6 +272,15 @@ def _init_device(self) -> None:
             self.device = torch.device("cuda", 0)
         assert self.device is not None, "Error setting torch device."
 
+    def _set_default_gradient_compression(self, gradient_compression: bool) -> None:
+        self._fp16_compression_default = gradient_compression
+
+    def _set_default_average_aggregated_gradients(self, average_aggregated_gradients: bool) -> None:
+        self._average_aggregated_gradients_default = average_aggregated_gradients
+
+    def _set_is_pre_trainer(self) -> None:
```

Comment:
this shouldn't be needed if we're not doing the external checkpoint loading thing